internal/field/internal/templates/element/conv.go

```diff
@@ -159,7 +159,6 @@ func (z *{{.ElementName}}) SetBigInt(v *big.Int) *{{.ElementName}} {
 	vv := bigIntPool.Get().(*big.Int)
 
 	// copy input + modular reduction
-	vv.Set(v)
```

Comment:
`vv` will be used as an output variable in the operation next line, so it shouldn't make